app/assets/stylesheets/splash.scss

```diff
@@ -0,0 +1,25 @@
+.splash_container {
+  // background-image: image-url('chalkboard.jpg');
+  // background-size: 100%;
+  // background-repeat: no-repeat;
+  background-color: #000000;
+  background-image: image-url('chalkboard.jpg');
+  background-size: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: fixed;
+  top: 0; right: 0; bottom: 0; left: 0;
```

Comment:
Property 'right' should be placed on own line<br>Property 'bottom' should be placed on own line<br>Property 'left' should be placed on own line